src/NServiceBus.Callbacks.Testing/NServiceBus.Callbacks.Testing.csproj

```
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+
+    <TargetFrameworks>net472;netcoreapp3.1;net6.0</TargetFrameworks>
```

@soujay Watch your diffs before commit for accidentally adding extra newlines or whitespace
```suggestion
    <TargetFrameworks>net472;netcoreapp3.1;net6.0</TargetFrameworks>
```